File: client/src/models/Person.ts
import { Name } from "./Name";

export class Person {
  name?: Name;
  cellPhoneNumber: string | null = null;
  emailAddress: string | null = null;
  iD: string | null = null;
  bio?: string | null = null;

  constructor(init?: Partial<Person>) {
    Object.assign(this, init);
    this.name = new Name(this.name);
  }
}

File: server/PersonalServer/Models/Person.cs
using System;

namespace PersonalServer.Models
{
    public class Person
    {
        public Guid Id { get; set; }
        public Name Name { get; set; }
        public string CellPhoneNumber { get; set; }
        public string EmailAddress { get; set; }

        public string Bio { get; set; }
    }
}
